tests/subsys/pm/power_mgmt/src/main.c

```diff
@@ -444,6 +450,36 @@ ZTEST(power_management_1cpu, test_device_state_lock)
 	testing_device_lock = false;
 }
 
+ZTEST(power_management_1cpu, test_empty_states)
+{
+	const struct pm_state_info *cpu_states;
+	uint8_t state = pm_state_cpu_get_all(1u, &cpu_states);
+
+	zassert_equal(state, 0, NULL);
+}
+
+ZTEST(power_management_1cpu, test_force_state)
+{
+	forced_state = PM_STATE_STANDBY;
+	bool ret = pm_state_force(0, &(struct pm_state_info) {forced_state, 0, 0});
+
+	zassert_equal(ret, true, "Error in force state");
+	set_pm = true;
```

Comment:
This should be set by the notifier in the other test. You don't need to manipulate it. Instead, just return after you check this test in `pm_state_set`.
```
	if (testing_force_state) {
		/* if forced to given power state was called */
		zassert_equal(state, forced_state, NULL);
		return;
	}
```

Reply:
Without this parameter set to true, test is failing on assert in pm_state_set : 
`DEBUG   - OUTPUT: START - test_force_state
DEBUG   - OUTPUT:     Assertion failed at WEST_TOPDIR/zephyr/tests/subsys/pm/power_mgmt/src/main.c:188: pm_state_set: (set_pm == true is false)
DEBUG   - OUTPUT: Notification to enter suspend was not sent to the App`